README.md

```diff
@@ -1,21 +1,52 @@
-# ``cleanlab`` Examples
+# cleanlab Examples
 
-Not sure where to start? Try checking out how to find [ImageNet Label Errors](imagenet/imagenet_train_label_errors.ipynb).
+This repo contains code examples that demonstrate how to use [cleanlab](https://github.com/cleanlab) and how [confident learning](https://arxiv.org/abs/1911.00068) works to find label errors.
 
+## Latest Examples
```

Comment:
"Table of Contents" may be better header?

"Latest Examples" makes it sound like this stuff is very new when it is not.

Reply:
That's true. "Table of Contents" sounds good to me. I made the change.